daemon/daemon.go

```diff
@@ -1195,9 +1195,10 @@ func NewDaemon(ctx context.Context, config *config.Config, pluginStore *plugin.S
 	engineMemory.Set(float64(info.MemTotal))
 
 	log.G(ctx).WithFields(log.Fields{
-		"version":     dockerversion.Version,
-		"commit":      dockerversion.GitCommit,
-		"graphdriver": d.ImageService().StorageDriver(),
+		"version":                dockerversion.Version,
+		"commit":                 dockerversion.GitCommit,
+		"storage-driver":         d.ImageService().StorageDriver(),
+		"containerd-snapshotter": d.UsesSnapshotter(),
```

Comment:
Looking for feedback on this one (also see my PR description); perhaps `type=io.containerd.snapshotter.v1` would be better?

Reply:
The `UsesSnapshotter` is a feature (i.e. the `features` property in the daemon.json file. Maybe it wouldbe better to log all the features the daemon has instead.

This will help in the future too if we create new features

Reply:
Hm.. so I tried adding the features, but that of course doesn't work when using the `TEST_INTEGRATION_USE_SNAPSHOTTER` env-var (unless we would merge that with the features map) 🤔

Reply:
I think the current version is fine, not sure `type=io.containerd.snapshotter.v1` if adds more information than `containerd-snapshotter=true`. It's not an API, so we can always change it in the future if we need to.

Reply:
> Hm.. so I tried adding the features, but that of course doesn't work when using the `TEST_INTEGRATION_USE_SNAPSHOTTER` env-var (unless we would merge that with the features map) 🤔

Maybe we should merge it since that env var overrides it